diff --git a/index.js b/index.js
index 0b9d100..c9dba96 100644
--- a/index.js
+++ b/index.js
@@ -1,11 +1,11 @@
 var Accessory = require('./lib/Accessory.js').Accessory;
 var Bridge = require('./lib/Bridge.js').Bridge;
-var Camera = require('./lib/Camera.js').Camera;
+//var Camera = require('./lib/Camera.js').Camera;
 var Service = require('./lib/Service.js').Service;
 var Characteristic = require('./lib/Characteristic.js').Characteristic;
 var uuid = require('./lib/util/uuid');
 var AccessoryLoader = require('./lib/AccessoryLoader.js');
-var StreamController = require('./lib/StreamController.js').StreamController;
+//var StreamController = require('./lib/StreamController.js').StreamController;
 var storage = require('node-persist');
 
 // ensure Characteristic subclasses are defined
@@ -15,12 +15,12 @@ module.exports = {
   init: init,
   Accessory: Accessory,
   Bridge: Bridge,
-  Camera: Camera,
+//  Camera: Camera,
   Service: Service,
   Characteristic: Characteristic,
   uuid: uuid,
   AccessoryLoader: AccessoryLoader,
-  StreamController: StreamController
+//  StreamController: StreamController
 }
 
 function init(storagePath) {
@@ -29,4 +29,4 @@ function init(storagePath) {
     storage.initSync({ dir: storagePath });
   else
     storage.initSync(); // use whatever is default
-}
\ No newline at end of file
+}
